<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDisplayWindow</class>
 <widget class="QDialog" name="EntryDisplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entry</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEditCreated">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yyyy hh:mm AP</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditUsername">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelCreated">
      <property name="text">
       <string>Created on</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="buttonShowHidePassword">
      <property name="text">
       <string>Show Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEditModified">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yyyy hh:mm AP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditTitle">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelPassword">
      <property name="text">
       <string>Password</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelModified">
      <property name="text">
       <string>Modified on</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Title</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelUsername">
      <property name="text">
       <string>Username</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="buttonClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="buttonSaveClose">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Keep and Close</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="buttonEditing">
      <property name="text">
       <string>Enable Editing</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelParent">
      <property name="text">
       <string>Parent</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboParent">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>dateTimeEditCreated</tabstop>
  <tabstop>dateTimeEditModified</tabstop>
  <tabstop>buttonShowHidePassword</tabstop>
  <tabstop>buttonEditing</tabstop>
  <tabstop>buttonSaveClose</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
